platformLibs/src/platform/osx/AGL.def.disabled

```diff
@@ -0,0 +1,6 @@
+language = Objective-C
+package = platform.AGL
+modules = AGL
+compilerOpts = -framework AGL
+linkerOpts = -framework AGL
+#Disabled: Framework without module
```

Comment:
I'm not sure it is the right reason. But IIRC, the framework is deprecated. Maybe mention this instead?

Reply:
Yeah, sure. Sadly, there is no obvious way to distinguish deprecated frameworks automatically.